aws/load_balancer/waf.tf

```
@@ -433,35 +517,44 @@ resource "aws_wafv2_web_acl_logging_configuration" "firehose_waf_logs_forms" {
 resource "aws_wafv2_regex_pattern_set" "valid_app_uri_paths" {
   name        = "valid_app_uri_paths"
   scope       = "REGIONAL"
-  description = "Regex to match the app and api valid urls"
+  description = "Regex to match the app valid urls"
 
+  # App paths
   regular_expression {
-    regex_string = "^\\/(?:en|fr)?\\/?(?:(admin|id|api|auth|signup|profile|forms|unsupported-browser|terms-of-use|contact|support|404)(?:\\/[\\w-]+)?)(?:\\/.*)?$"
+    regex_string = "^\\/(?:en|fr)?\\/?(?:(admin|form-builder|forms|id|auth|profile|support|contact|unlock-publishing)(?:\\/[\\w-]+)?)(?:\\/.*)?$"
   }
 
+  # Static Pages
   regular_expression {
-    regex_string = "^\\/(?:en|fr)?\\/?(?:(form-builder|sla|unlock-publishing|terms-and-conditions|javascript-disabled)(?:\\/[\\w-]+)?)(?:\\/.*)?$"
+    regex_string = "^\\/(?:en|fr)?\\/?(?:(sla|terms-and-conditions|terms-of-use|unsupported-browser|javascript-disabled|404)(?:\\/[\\w-]+)?)(?:\\/.*)?$"
   }
 
+  # Files
   regular_expression {
     regex_string = "^\\/(?:en|fr)?\\/?(?:(static|_next|img|favicon\\.ico)(?:\\/[\\w-]+)*)(?:\\/.*)?$"
   }
 
-  # API paths
-  regular_expression {
-    regex_string = "^\\/(?:v1)?\\/?(?:(docs|status))(?:\\/)?$"
-  }
-
   # This is a temporary rule to allow search engines tools to access ownership verification files
   regular_expression {
     regex_string = "^\\/?(BingSiteAuth\\.xml|googlef34bd8c094c26cb0\\.html)$"
   }
 
+  # Language selector page
   regular_expression {
     regex_string = "^\\/(?:en|fr)?\\/?$"
   }
 }
 
+resource "aws_wafv2_regex_pattern_set" "valid_api_uri_paths" {
+  name        = "valid_api_uri"
+  scope       = "REGIONAL"
+  description = "Regex to match the api valid urls"
+
+  regular_expression {
+    regex_string = "^(?:\\/v1)?\\forms\\/(?:(c[a-z0-9]{24}))\\/(?:(template|(?:(submission\\/(?:(new|(?:([0-9]{2}-[0-9]{2}-[a-z0-9]{4})\\/?(?:(confirm\\/[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89ab][a-f0-9]{3}-[a-f0-9]{12}|problem)?))))))))(?:\\/)?$"
```

Forgot to mention this with my previous comments:

```suggestion
    regex_string = "^(?:\\/v1)?\\/forms\\/(?:(c[a-z0-9]{24}))\\/(?:(template|(?:(submission\\/(?:(new|(?:([0-9]{2}-[0-9]{2}-[a-z0-9]{4})\\/?(?:(confirm\\/[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89ab][a-f0-9]{3}-[a-f0-9]{12}|problem)?))))))))(?:\\/)?$"
```